dss/api/bundles/checkout.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+import boto3
+import os
+import requests
+from flask import jsonify
+from cloud_blobstore import BlobNotFoundError
+from dss import Config
+
+from dss.util.bundles import get_bundle
+from dss.util.checkout import get_execution_id
+from ... import dss_handler, get_logger
+
+
+@dss_handler
+def post(uuid: str, json_request_body: dict, replica: str, version: str = None):
+
+    sfn = boto3.client('stepfunctions')
+    sts_client = boto3.client("sts")
+
+    email = json_request_body['email']
+    dss_bucket = Config.get_s3_bucket()
+
+    if replica is None:
```

Comment:
Isn't `replica` mandatory?

Reply:
yes, I added an assertion

Reply:
I don't think you need to.  connexion should verify it if `required` is True in the swagger spec.